### PR TITLE
environment variables

### PR DESCRIPTION
**Pull request by kavi-i**

### CODE REVIEW

File: vmailSeneca/config/default.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	  "secure": true
 	},
   	"auth": {
```
Comment (chiragob) [inline vmailSeneca/config/default.json]: remove email id

File: vmailServer/config/default.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
     "db": "virtualEMail",
     "servers": [
       {
-        "host": "localhost",
-        "port": "28015"
+        "host": "139.59.35.45",
```
Comment (chiragob) [inline vmailServer/config/default.json]: set localhost

File: vmailServices/email-services/config/default.json [hunk 3]
```diff
@@ -1,8 +1,8 @@
 {
-  "rhost": "localhost",
-  "rport": "28015",
+  "rhost": "139.59.35.45",
```
Comment (chiragob) [inline vmailServices/email-services/config/default.json]: set localhost

**Review by chiragob — COMMENTED** [review]
update it

File: vmailSeneca/config/default.json (hunk 1 above)
Comment (kavi-i) [reply vmailSeneca/config/default.json]: done

File: vmailServer/config/default.json (hunk 2 above)
Comment (kavi-i) [reply vmailServer/config/default.json]: done

File: vmailServices/email-services/config/default.json (hunk 3 above)
Comment (kavi-i) [reply vmailServices/email-services/config/default.json]: done

**Review by chiragob — APPROVED** [review]
done

**Pull request merged**